vm-validator/src/vm_validator.rs

```diff
@@ -34,11 +34,11 @@ pub struct VMValidator {
 
 impl VMValidator {
     pub fn new(
```

Comment:
Delete `config` parameter + refactor clients of `VMValidator::new`?

Reply:
I was a bit hesitated to do that because VMValidator might want to store other configs in the future.

Reply:
Let's clean up unused params now and let someone add them back if they're needed in the future?